### ISSUE TITLE
Population module crashes due to empty x.S.TissueMasking

### ISSUE THREAD

**Issue by BeatrizPadrela** [ExploreASL/ExploreASL#1857]
### Description

This crashes when running Population module with specifying TissueMasking. And gives an error right after saying it will use default values:

<img width="1025" alt="image" src="https://github.com/user-attachments/assets/eff04184-d97b-4d75-8364-703dda82cded" />

Then, even if I comment this error line, it still crashes here because x.S.Atlases length is higher than x.S.TissueMasking:
<img width="906" alt="image" src="https://github.com/user-attachments/assets/006d3a56-215c-4afb-a8c1-55f46f623f6d" />

### Release notes
Nothing. This is a true input error. The error messages were previously improved.

**Comment by HenkMutsaerts** [comment]
Is this duplicate with #1801?

**Comment by jan-petr** [comment]
This is correct. It should crash if you provide incorrect atlases+tissue masking. 

The message was previously improved and is now pretty informative. So duplicate indeed.
